app/views/advanced/_guided_search_fields.html.erb

```diff
@@ -1,6 +1,7 @@
 
 <div class="advanced-search-field">
   <div class="search-field">
+    <label class="sr-only">Options for advanced search</label>
     <%= select_tag('f1', options_for_select(advanced_key_value, guided_field(:f1, 'all_fields')), :class=>"search_field", 'aria-label': 'Options for advanced search') %>
```

Comment:
We can remove the `aria-label` attribute and value since we are replacing it with the label element.

Reply:
will do, thanks!